repose-aggregator/tests/performance-tests/test_vars/filters/scripting.yml

```diff
@@ -1,3 +1,8 @@
+# TODO: Remove this file? Don't want it to get picked up by the automated performance tests, but
+# TODO: keeping this file gives us a sensibly named way to run generic scripting tests.
+#
+# TODO: YAML does not seem to support include or inherit. As such, there is more duplication than
+# TODO: necessary because each scripting language only changes the "script_lang" variable.
```

Comment:
I'm not sure I understand the case for keeping this file. If we delete it, we're talking about going from this:

```
ansible-playbook site.yml \
  -e '{perf_test: filters/scripting,
       script_lang: python}'
```

to this:

```
ansible-playbook site.yml \
  -e '{perf_test: filters/scripting/python}'
```

right?

I suppose someone might do something ridiculous like:

```
ansible-playbook site.yml \
  -e '{perf_test: filters/scripting/groovy,
       script_lang: python}'
```

Which we can't stop, but either way I'm fine with deleting this file.